content/docs/integrations.mdx

```diff
@@ -78,28 +78,42 @@ The Pomerium Databroker would be updated with types `example.com/geoip`, `device
 
 Compressed versions are supported using `gz` format.
 
-### **Record Type**
+### Record Type
 
 Unless defined by the directory structure of a supplied archive file, the Record Type field defines how the records will be stored and accessed in the Databroker.
 
-### **Foreign Key**
+### Foreign Key
 
 **Foreign Key** is used to map an authorization evaluation to the corresponding record. The supported values are:
 
-- `user.id` (Also the default if no value is provided),
-- `user.email`,
-- `request.ip`,
-- `device.id`.
+- `user.id` (Also the default if no value is provided)
+- `user.email`
+- `request.ip`
+- `device.id`
 
-### **Headers**
+#### IP range lookup support
+
+Pomerium supports Classless Inter-Domain Routing (CIDR) indexing when filtering a list of records with the `request.ip` foreign key. This allows you to match against IP address ranges instead of manually importing records for individual IP addresses.
```

Comment:
"Pomerium supports Classless Inter-Domain Routing (CIDR) indexing" sounds off to me. "Classless Inter-Domain Routing" has to do with network architecture. "CIDR notation" is the operative term here.

I understand the desire to spell out any potentially-unfamiliar acronyms, but here I think it would tend to increase confusion rather than decrease it.

Would a link to wikipedia help to address this concern in a different way?

```suggestion
For the `request.ip` foreign key, Pomerium also supports matching against a range of IP addresses (expressed in [CIDR notation](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing#CIDR_notation)). This can reduce the number of data records you need.
```